elastic/beats
# Implements network.tos in Packetbeat

This is an enhancement to Packetbeat.

## What does this PR do?

It implements export of network ToS (Type of Service) and closes #29068

## Why is it important?

Because Pakcetbeat is currently blind to ToS that instead is a popular metric used to mark network traffic

## Checklist

- [ X] My code follows the style guidelines of this project
- [ X] I have commented my code, particularly in hard-to-understand areas
- [ X] I have made corresponding changes to the documentation


## How to test this PR locally

Fire up packet beat and check in Elastic that new record entry is added. (see below)

## Related issues

Closes #29068

## Logs

In JSON you will see a new block such as

"network.tos": [
      0
    ],
